Makefile

```diff
@@ -143,6 +143,10 @@ controller-tests: kube-apiserver etcd kubectl
 	TEST_ASSET_KUBECTL=$(KUBECTL_BIN) \
 	go test -test.v ./pkg/controllers/... -controller-test
 
+.PHONY: gadgets-unit-tests
+gadgets-unit-tests:
+	go test -test.v -exec sudo ./pkg/gadgets/...
```

Comment:
By the way, we should use this approach in other tests, e.g., local-gadget-tests.